lib/gli/commands/scaffold.rb

```diff
@@ -31,6 +31,7 @@ def self.create_scaffold(root_dir,
           end
           puts "Created #{rvmrc}"
         end
+        system("git", "init", "--quiet", "#{root_dir}/#{project_name}")
```

Comment:
Cool.  One other thing - I think we should check the return value and give a message if things go wrong.  Probably something simple like

```ruby
unless system(...)
  puts "Problem initializing this as a git repo - your gemspec may not work until you have done a successful `git init`"
end
```

Reply:
Done. Feel free to edit the message, as english is not my natural language.